R/get_connection.R

```diff
@@ -172,15 +172,29 @@ get_nuvolos_db_path <- function() {
 #' 
 #' Using outside of Nuvolos only. This helps the user to store credentials safely at local device.
 #' @export
-input_nuvolos_credential <- function(username, password){
+input_nuvolos_credential <- function(){
```

Comment:
I think we should remove this whole part and use rstudioapi::askForSecret("Please input your Nuvolos username:") directly in the connection setup. So no need for a separate 'config setup', instead the user can simply click "save in keyring" for themselves.

Reply:
but what if they input a wrong one, and need to change it...

Reply:
I would assume that is handled by Rstudio :)

Reply:
No, I mean if the user inputs the credential incorrectly, the keyring will save it anyway.  Then the connection will always fail without an approach to reinput... Rstudio couldn't handle that either...